.github/ISSUE_TEMPLATE/content-report.md

```diff
@@ -1,9 +1,8 @@
 ---
 name: Content Report
-about: Submit a Content Report if you find something wrong with the content within a page, ie Copyrights, Grammar, False Information or broken links to academic sources! For functional issues (such as our website not working), please use a Bug Report! 
-  its explained).
-title: ''
-labels: 'Content: Issue'
+about: " ie Copyrights, Grammar, False Info, broken links! "
```

Comment:
```suggestion
about: "This could be things like copyright issues, grammatical errors, false/incorrect information, broken links/markup."
```
Minor tweak to the stub to make it a little more descriptive as to what goes here.